lib/elixir/lib/file.ex

```diff
@@ -1612,8 +1612,8 @@ defmodule File do
 
   The `line_or_bytes` argument configures how the file is read when
   streaming, by `:line` (default) or by a given number of bytes. When
-  using the `:line` option, CRLF line breaks (`"\r\n"`) are normalized
-  to LF (`"\n"`).
+  using the `:line` option, CRLF line breaks (`"\\r\\n"`) are normalized
+  to LF (`"\\n"`).
```

Comment:
Instead of this, to avoid others falling into this "gotcha" in the future, you can change this particular docstring to use the `~S` sigil. You can do that by changing line 1607 (a few lines above) from `@doc """` to `@doc ~S"""`. 🙃

Reply:
Great idea! Thanks @whatyouhide!